README.md

```diff
@@ -385,24 +385,46 @@ libraryJob.run()
         libraryJob.sh(curl -H 'Content-Type: application/json' -X POST -d '{"name" : "Ben"}' http://acme.com)
 ```
 
+## Declarative Pipeline Support
+
+Declarative pipelines are supported via a test harness.  The tests use closures in the file as units of isolation.  Effectively,
+your tests describe which part of the jenkinsfile your test is verifying and don't interract with the rest of the file.  Here is an example that
+verifies that sh was invoked during the "init" stage.
+
+```
+@Test
+void runAStage() {
+    super.jenkinsfile.run()
+    harness
+        .runStage('init')
+        .runClosureStep('steps')
```

Comment:
@dgoetsch This looks cool :+1:

Is there a way to just run an entire declarative pipeline in a similar fashion to Jenkins too?

Reply:
I believe this is relevant to the comment from @stchar asking about built in support for `parameters` / `environment` / `pre` / `post`.